### ISSUE TITLE
ashell access from files app

### ISSUE THREAD

**Issue by ifuchs** [holzschu/a-shell#378]
I can transfer files to/from ashell using the Shortcut actions but is there a way to expose ashell's file directory to the files app?

**Comment by holzschu** [comment]
`~/Documents` is visible in the Files app, under "On my iPad/a-Shell".

**Comment by ifuchs** [comment]
That’s what I thought but I don’t see it on my iPad or iPhone. Do I have to do anything to create it?

**Comment by holzschu** [comment]
No, except maybe wait a little or reboot the iPad/iPhone. Try if it appears from a-Shell using pickFolder.
